experimental/dds/tree2/src/test/shared-tree-core/editManagerTestUtils.ts

```diff
@@ -43,34 +53,36 @@ export function editManagerFactory(options: {
 			revertible.discard();
 		});
 	}
-
-	return { manager, family };
+	return manager;
 }
 
-export function rebaseLocalEditsOverTrunkEdits(
+export function rebaseLocalEditsOverTrunkEdits<TChange>(
```

Comment:
preexisting, but do we even get anything from overloading here and below? We're not documenting the different overloads with different doc comments, and just using an optional param with default `false` seems about as clear.

Reply:
We do get something out of it: if you pass the `defer` argument then you're guaranteed to get the `() => void` back, and if you don't pass it, then you're guaranteed not to. Adding doc comments to help.